test/commands.test.ts

```diff
@@ -742,41 +740,38 @@ suite('Command Handler', () => {
             await handler.restartServer('run');
 
             // then
+            expect(stubs.incoming.onServerStateChanged).calledBefore(stopStub);
             expect(stopStub).calledOnceWith(false, ProtocolStubs.startedServerState);
-            expect(startStub).calledAfter(stopStub);
-            expect(startStub).calledOnceWith('run', ProtocolStubs.startedServerState);
         });
 
-        test('error if mode doesn\'t contains a valid value', async () => {
-            try {
-                await handler.restartServer('fakeMode', ProtocolStubs.startedServerState);
-                expect.fail();
-            } catch (err) {
-                expect(err).equals('Could not restart server: unknown mode fakeMode');
-            }
-        });
+        // test('error if mode doesn\'t contains a valid value', async () => {
```

Comment:
this test seem useful, it asserts what happens if the mode is something that differs from 'debug' or 'run'. Why commenting it?